doc/src/kspace_tild.rst

```diff
@@ -234,41 +220,41 @@ The current interaction styles used in *tild cross-interaction* are
    U_{g-erfc} = & \frac{A}{\rho_0} u_G (r) * \text{erfc}
    \left(\frac{\vert r \vert - R_p}{\xi}\right)
 
-where :math:`A` is the value set by `tild prefactor`\ , :math:`\rho_0` is the TILD density of the simulation box, :math:`\sigma` is the gaussian width, :math:`R_p` is the erfc particle radius and :math:`\xi` is the erfc width, which controls how quickly the particle density drops from :math:`\rho_0`` to zero.
+where :math:`A` is the value set by `tild prefactor`\ , :math:`\rho_0` is the TILD density of the simulation box, :math:`\sigma` is the gaussian width, :math:`R_p` is the erfc particle radius and :math:`\xi` is the erfc width, which controls how quickly the particle density drops from :math:`\rho_0`` to zero. :math:`U_{g-erfc}` involves convoluting the :math:`U_{g}` and :math:`U_{erfc}` functions.
 
 The first required keyword for the *tild cross-interaction* option is the interaction model. 
 Currently supported options for interaction models
 and their required arguments are:
 
 1. *gaussian* : :math:`\sigma` (distance units)
-2. *gaussian-erfc* : :math:`\sigma`\ , :math:`R_p`, :math:`\xi` (all in distance units)
+2. *gaussian_erfc* : :math:`\sigma`\ , :math:`R_p`, :math:`\xi` (all in distance units)
 3. *erfc* : :math:`R_p`\ , :math:`\xi` (both in distance units)
```

Comment:
As you say in the note, we ask them for sigma squared, not sigma.  That should probably be reflected in this part too, for example

```suggestion
1. *gaussian* : :math:`\sigma^2` (distance:math:`^2` units)
 2. *gaussian_erfc* : :math:`\sigma^2`\ , :math:`R_p`, :math:`\xi` (distance units except for :math:`\sigma^2` )
 3. *erfc* : :math:`R_p`\ , :math:`\xi` (both in distance units)
```

Reply:
In the code, it takes in the sigma and squares the value. See line 1560. I'm open to a discussion about whether that should change.

Reply:
I thought line 1560 is called with `tild shape`. For `tild cross-interaction` `calc_cross_work` assumes it is squared, right?